### PR TITLE
Update config file for anmn_hourly_timeseries

### PR DESCRIPTION
**Pull request by xhoenner**
As discussed by email @lbesnard

### CODE REVIEW

File: aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json [hunk 1]
```diff
@@ -1,22 +1,16 @@
 {
   "dataset_name": "anmn_hourly_timeseries",
-  "logger_name": "anmn_hourly_timeseries",
```
Comment (lbesnard) [inline aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json:3]: a bit annoying that the order changed, it's harder to review and see the changes

File: aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
+  "logger_name": "anmn_hourly_timeseries",
+  "gattrs_to_variables" : ["site_code"],
+  "partition_keys": ["site_code", "timestamp", "polygon"],
   "time_extent": {
     "time": "TIME",
     "partition_timestamp_period": "Q"
   },
   "spatial_extent": {
-    "lat": "LATITUDE",
+    "lat":  "LATITUDE",
```
Comment (lbesnard) [inline aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json:13]: two spaces

File: aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json [hunk 3]
```diff
@@ -1,22 +1,16 @@
...
-    "site_code"
-  ],
-  "partition_keys": [
-    "site_code",
-    "timestamp",
-    "polygon"
-  ],
+  "logger_name": "anmn_hourly_timeseries",
+  "gattrs_to_variables" : ["site_code"],
+  "partition_keys": ["site_code", "timestamp", "polygon"],
```
Comment (lbesnard) [inline aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json:7]: again, annoying that this changed, not a deal breaker, but makes seeing the actual changes way harder

**Comment by xhoenner** [comment]
Thanks for the review @lbesnard, and apologies about the messy PR. Should be cleaner now ^_^

**Pull request closed without merging**